src/shared/Microsoft.AzureRepos.Tests/AzureReposHostProviderTests.cs

```diff
@@ -182,17 +182,15 @@ public async Task AzureReposHostProvider_ConfigureAsync_UseHttpPathSetTrue_DoesN
             var provider = new AzureReposHostProvider(new TestCommandContext());
 
             var environment = new TestEnvironment();
-            var config = new TestGitConfiguration(new Dictionary<string, IList<string>>
-            {
-                [AzDevUseHttpPathKey] = new List<string> {"true"}
-            });
+            var git = new TestGit();
+            git.GlobalConfiguration.Dictionary[AzDevUseHttpPathKey] = new List<string> {"true"};
 
             await provider.ConfigureAsync(
                 environment, EnvironmentVariableTarget.User,
-                config, GitConfigurationLevel.Global);
+                git, GitConfigurationLevel.Global);
 
-            Assert.Single(config.Dictionary);
-            Assert.True(config.Dictionary.TryGetValue(AzDevUseHttpPathKey, out IList<string> actualValues));
+            Assert.Single(git.GlobalConfiguration.Dictionary);
+            Assert.True(git.GlobalConfiguration.Dictionary.TryGetValue(AzDevUseHttpPathKey, out IList<string> actualValues));
```

Comment:
I'm torn between if I like fluent assertions or not. The syntax can be harder to read IMO. Which bits are exercising code, and which bits are verifying results or state? Perhaps I'm just jaded because, again, I've seen some absolutely terrible use of fluent assertions in other codebases.

Reply:
Interesting, in most of the code bases we have on devplat it's vastly improved readability of our assertions (in our teams opinion). If you have any callouts of bad uses I'd love to take a look.